src/Integration/Binding/DotNetBindingConfigProvider.cs

```diff
@@ -154,5 +161,16 @@ private static VsRuleset ToVsRuleset(CoreRuleset coreRuleset)
             
             return ruleSet;
         }
+
+        internal static  /* for testing */ bool IsSupportedRule(SonarQubeRule rule)
+        {
+            // We don't want to generate configuration for taint-analysis rules or hotspots.
+            // * taint-analysis rules: these are in a separate analyzer that doesn't ship in SLVS so there is no point in generating config
+            // * hotspots: these are noisy so we don't want to run them in the IDE. There is special code in the Sonar hotspot analyzers to 
+            //              control when they run; we are responsible for not generating configuration for them.
+
+            // duncanp: TODO - exclude hotspots. The SonarQubeRule doesn't currently contain sufficient data for us to do this.
```

Comment:
This PR just moves the existing filtering logic to a central location. I'll add the hotspot filtering next, although I suspect I might need to change the client to return additional information.